dags/sentinel-1_indexing.py

```diff
@@ -0,0 +1,173 @@
+    Secret("env", "DB_DATABASE", "explorer-writer", "database-name"),
+]
+
+INDEXER_IMAGE = "opendatacube/datacube-index:0.0.16"
+OWS_IMAGE = "opendatacube/ows:1.8.2"
+EXPLORER_IMAGE = "opendatacube/explorer:2.2.3"
+
+affinity = {
+    "nodeAffinity": {
+        "requiredDuringSchedulingIgnoredDuringExecution": {
+            "nodeSelectorTerms": [
+                {
+                    "matchExpressions": [
+                        {
+                            "key": "nodetype",
+                            "operator": "In",
+                            "values": [
+                                "ondemand",
+                            ],
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+
+OWS_BASH_COMMAND = [
+    "bash",
+    "-c",
+    dedent(
+        f"""
+        mkdir -p /env/config;
+        curl -s https://raw.githubusercontent.com/digitalearthafrica/config/0.1.5/services/deafrica_prod_af.ows_cfg.py --output /env/config/ows_cfg.py;
+        datacube-ows-update --views;
+        datacube-ows-update {PRODUCT_NAME};
+    """
+    ),
+]
+
+dag = DAG(
+    "Sentinel-1_Indexing",
+    doc_md=__doc__,
+    default_args=DEFAULT_ARGS,
+    schedule_interval="0 */1 * * *",
+    catchup=False,
+    tags=["k8s", "Sentinel-1"],
+)
+
+with dag:
+    INDEXING = KubernetesPodOperator(
+        namespace="processing",
+        image=INDEXER_IMAGE,
+        image_pull_policy="Always",
+        arguments=[
+            "sqs-to-dc",
+            "--stac",
+            "--update-if-exists",
+            "--allow-unsafe",
+            QUEUE_NAME,
+            PRODUCT_NAME,
+        ],
+        labels={"step": "sqs-to-rds"},
+        name="datacube-index",
+        task_id="indexing-task",
+        get_logs=True,
+        affinity=affinity,
+        is_delete_operator_pod=True,
+    )
+
+    # OWS_UPDATE_EXTENTS = KubernetesPodOperator(
+    #     namespace="processing",
+    #     image=OWS_IMAGE,
+    #     arguments=OWS_BASH_COMMAND,
+    #     secrets=OWS_SECRETS,
+    #     labels={"step": "ows-mv"},
+    #     name="ows-update-extents",
+    #     task_id="ows-update-extents",
+    #     get_logs=True,
+    #     affinity=affinity,
+    #     is_delete_operator_pod=True,
+    # )
+
+    # EXPLORER_SUMMARY = KubernetesPodOperator(
+    #     namespace="processing",
+    #     image=EXPLORER_IMAGE,
+    #     arguments=[
+    #         "cubedash-gen",
+    #         "--no-init-database",
+    #         "--refresh-stats",
+    #         "--force-refresh",
+    #         PRODUCT_NAME,
+    #     ],
+    #     secrets=EXPLORER_SECRETS,
+    #     labels={"step": "explorer"},
+    #     name="explorer-summary",
+    #     task_id="explorer-summary-task",
+    #     get_logs=True,
+    #     affinity=affinity,
+    #     is_delete_operator_pod=True,
+    # )
+
+    INDEXING
+    # INDEXING >> OWS_UPDATE_EXTENTS
+    # INDEXING >> EXPLORER_SUMMARY
```

Comment:
This commented block is placeholder for explore and ows so it's fine.

Reply:
👍